app.py

```diff
@@ -65,6 +71,19 @@ def get_hourly_forecast():
     response = forecast_service.get_hourly_forecast(request_model)
     return jsonify(response.dict()), response.status_code
 
+@app.route('/geocode/<string:zipcode>', methods=['GET'])

```

Comment:
We don't need an endpoint for this because the front-end will never directly call the geocoding. That will be the responsibility of the back-end